spec/models/response_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Response, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
this was created automatically when I created the model, haven't touched it yet. Not sure what to test yet since there isn't really anything in the model yet. Will most likely take this out and add our own stuff in later.

pending works like this:
`describe "an example" do
  it "is implemented but waiting" do
    pending("something else getting finished")
    this_should_not_get_executed
  end
end`

they just gave it to us as an outline though, I've never actually used pending in a spec before tbh